doc/troubleshooting.rst

```diff
@@ -3,14 +3,19 @@ Troubleshooting
 
 Here you can find some tips to try to find out why your installation of NEST didn't work.
 
+Please make sure you have:
 
-1. CMAKE error says a <package> was not found or <package> is too old
----------------------------------------------------------------------
-
+* Followed the :doc:`installation instructions <installation/index>`
+* Installed the required :ref:`dependencies <standard>`
+* Viewed the :doc:`compilation <installation/compilation_options>` and 
+  :doc:`configuration <installation/configuration>` options
```

Comment:
In my view the items after the bullet points should all start with a lower case letter and the final one should end in a full stop. But as we seem to use the same form as here throughout the whole documentation, I don't think this PR is the place to change anything but I rather wanted to take the opportunity to voice my opinion. Feel free to ignore it :wink:

Reply:
I agree with you here! However, for the sake of consistency with the remaining documentation, I would keep it as it is for now. We can address the style in a separate issue  :+1:.